src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] You'll kindly also need to make this component(App) class-based. This requirement is essential :+1:  for the approval :heavy_check_mark:  of this milestone.